README.md

```diff
@@ -29,7 +29,7 @@ yarn add imagekitio-react
 Import components in your code:
 
 ```js
-import { IKImage, IKContext, IKUpload } from 'imagekitio-react'
+import { IKImage, IKVideo, IKContext, IKUpload } from 'imagekitio-react'
 ```
 
```

Comment:
Shouldn't we also include this section - https://github.com/imagekit-developer/imagekit-javascript#access-request-id-other-response-headers-and-http-status-code

Reply:
Added here: https://github.com/imagekit-developer/imagekit-react/pull/93/commits/ce6dec5e4afc010d6083d5b54fa192c5fb6a2367